time,level,logger,message,callsite,exception
2020-05-02 14:05:04.6994,Error,Facelift_App.Repositories.ShipmentRepository,Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.,Facelift_App.Repositories.ShipmentRepository.CreateAccidentReportAsync,"System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.Internal.LazyInternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync()
   at Facelift_App.Repositories.ShipmentRepository.<CreateAccidentReportAsync>d__24.MoveNext() in D:\Duta Kalingga Pratama\Projects\Unilever\Facelift\Application\UNILEVER_Facelift\Facelift_App\Repositories\ShipmentRepository.cs:line 837"
2020-05-02 14:05:50.5351,Error,Facelift_App.Repositories.ShipmentRepository,Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.,Facelift_App.Repositories.ShipmentRepository.CreateAccidentReportAsync,"System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.Internal.LazyInternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync()
   at Facelift_App.Repositories.ShipmentRepository.<CreateAccidentReportAsync>d__24.MoveNext() in D:\Duta Kalingga Pratama\Projects\Unilever\Facelift\Application\UNILEVER_Facelift\Facelift_App\Repositories\ShipmentRepository.cs:line 837"
2020-05-02 14:09:19.5263,Error,Facelift_App.Repositories.ShipmentRepository,Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.,Facelift_App.Repositories.ShipmentRepository.CreateAccidentReportAsync,"System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.Internal.LazyInternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync()
   at Facelift_App.Repositories.ShipmentRepository.<CreateAccidentReportAsync>d__24.MoveNext() in D:\Duta Kalingga Pratama\Projects\Unilever\Facelift\Application\UNILEVER_Facelift\Facelift_App\Repositories\ShipmentRepository.cs:line 837"
2020-05-02 14:13:04.4352,Error,Facelift_App.Repositories.ShipmentRepository,Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.,Facelift_App.Repositories.ShipmentRepository.CreateAccidentReportAsync,"System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.Internal.LazyInternalContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync(CancellationToken cancellationToken)
   at System.Data.Entity.DbContext.SaveChangesAsync()
   at Facelift_App.Repositories.ShipmentRepository.<CreateAccidentReportAsync>d__24.MoveNext() in D:\Duta Kalingga Pratama\Projects\Unilever\Facelift\Application\UNILEVER_Facelift\Facelift_App\Repositories\ShipmentRepository.cs:line 837"
